drivers/i2c/i2c_dw.c

```diff
@@ -839,14 +853,17 @@ static int i2c_dw_runtime_configure(const struct device *dev, uint32_t config)
 
 			dw->hcnt = value;
 		} else {
-			rc = -EINVAL;
+			ret = -EINVAL;
```

Comment:
This change looks unrelated to the actual, functional change. Please leave this kind of changes out as it makes reading the diff much harder.